content/libraries/klc_history.adoc

```diff
@@ -5,6 +5,12 @@ url = "/libraries/klc/history/"
 +++
 
 ---
+== 3.0.21 - 2019-08-08
+* Clarified KiCad footprint assignment work-flows as generic vs fully specified symbols (G2.1)
+  * Avoid using the word atomic as it does not really specify what is really used.
```

Comment:
Perhaps change the final word, 'used' to 'meant'? Also, this is the only bullet to end with a period.

Reply:
completely reworded it instead of changing only one word